alembic/versions/eaca44fbeabe_add_soft_delete_to_events.py

```diff
@@ -0,0 +1,28 @@
+"""add soft delete to events
+
+Revision ID: eaca44fbeabe
+Revises: 404358628973
+Create Date: 2019-02-06 07:00:32.174447
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'eaca44fbeabe'
+down_revision = '404358628973'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('events', sa.Column('state', sa.Enum('active', 'archived', 'deleted', name='statetype'), nullable=True))
```

Comment:
This may also need to change so that already existing events have a default state of `active`.
You can refer to the `e132e132ed54_add_soft_delete.py` alembic file on how this is done.

Reply:
The difference is that the table initially wasn't being populated, but it a good fix...let's add that to make it foolproof. Thanks @RonKbS

Reply:
All your requests have been implemented @RonKbS

Reply:
Great catch @RonKbS